packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -551,10 +495,7 @@ class VMServiceFlutterDriver extends FlutterDriver {
   @override
   Future<void> forceGC() async {
     try {
-      await _peer
-          .sendRequest(_collectAllGarbageMethodName, <String, String>{
-        'isolateId': 'isolates/${_appIsolate.numberAsString}',
-      });
+      await _serviceClient.callMethod(_collectAllGarbageMethodName, isolateId: _appIsolate.id);
```

Comment:
I believe this method is deprecated and no longer does anything

Reply:
We're "not supposed" to use it, but it does work. It's not guaranteed to do anything but it normally works well enough.

We can't remove this - it's used in many of our benchmarks to reduce flakiness.